### ISSUE TITLE
using `.def()` with `oneOfType` throws error

### ISSUE THREAD

**Issue by donaldma** [dwightjack/vue-types#187]
When using `.def()` along with `oneOfType` it throws an error if the default value is not of the first type defined in `oneOfType`.

### Example:

works (string before number): 
```
w: oneOfType([string(), number()]).def('auto')
```
does not work (number before string):
```
w: oneOfType([number(), string()]).def('auto')
```

### Error:

![Screen Shot 2021-10-05 at 4 11 18 PM](https://user-images.githubusercontent.com/26336687/136115806-bfc0f3d5-4067-4bbb-b355-b903cddbbe30.png)

**Comment by dwightjack** [comment]
@donaldma I've published a fix in `4.1.1-rc.1`. Can you confirm the issue is solved?

**Comment by dwightjack** [comment]
Released in the stable channel as [v4.1.1](https://github.com/dwightjack/vue-types/releases/tag/4.1.1).
